test/utils/scene-graph/test-graph-theme-generator.js

```diff
@@ -144,5 +144,7 @@ describe('GraphThemeGenerator', function () {
                 })
             })
         });
+
+        // APEP TODO test to use a sceneGraph with incorrect type and make sure it doesn't do this auto mapping rule
```

Comment:
The check happens before the generators for the sound graph are triggered the the HubClient

Reply:
Cool - we should be OK to remove this TODO or just add as comment where it is checked :)